packages/@o3r/schematics/schematics/ng-add/index.ts

```diff
@@ -12,7 +12,7 @@ function ngAddFn(options: NgAddSchematicsSchema): Rule {
   const schematicsDependencies = ['@angular-devkit/architect', '@angular-devkit/schematics', '@angular-devkit/core', '@schematics/angular', 'globby'];
   return async (_, context) => {
     const packageJsonPath = path.resolve(__dirname, '..', '..', 'package.json');
-    const { getExternalDependenciesVersionRange, setupDependencies } = await import('@o3r/schematics');
+    const { getExternalDependenciesVersionRange, setupDependencies } = await import('../../src/public_api');
```

Comment:
What issue is fixing this change?
Does it still need to be dynamic?

Reply:
Hi Kilian, this is a draft to test if it could fix the following error:
![image](https://github.com/user-attachments/assets/98474b0d-6070-468d-b7cd-1658b5a7a3b3)

It is still WIP ;)

Reply:
usually when we are in `schematics/builders` and we need to import from the package itself, we use the name of the package instead of the relative path. This is because the builds are done separately and the build of `schematics/builders` is usually after the build of `src` and it can override the output of `src` if the import is relative.
If we do this change we need to check that the output of the `src` build is not broken

Reply:
I will take it into account but even the ng add @o3r/schematics does not work today:
![image](https://github.com/user-attachments/assets/3d01a5e2-1c00-41f5-a95e-c3ac0ffa564f)

Reply:
@mrednic-1A I just tested - the schematics build is composite and won't modify the result of the compile task if it is already available 
If I run the compile, then modify the dist/src/index and run only the build-builder, I keep my modifications.
So I think it would be okay

Reply:
I think we can conclude it reply to the first question.
I think we are still missing the second:

> Does it still need to be dynamic?